src/injectHighlightWrappers.spec.ts

```diff
@@ -47,11 +53,17 @@ describe('inject highlight wrappers for figure with caption', () => {
         startOffset: 2,
       };
       const highlight = new Highlight(range, highlightData, { formatMessage: jest.fn() });
+      highlight.getMessage = (id: string) => {
```

Comment:
instead of monkeypatching `highlight.getMessage`, you can pass a mock implementation into `formatMessage: jest.fn()`